mdsuite/calculators/einstein_diffusion_coefficients.py

```diff
@@ -184,17 +184,22 @@ def ensemble_operation(self, ensemble):
         msd = tf.math.squared_difference(
             tf.gather(ensemble, self.args.tau_values, axis=1), ensemble[:, None, 0]
         )
+        self.count += msd.shape[0]
         # average over particles, sum over dimensions
-        msd = tf.reduce_sum(tf.reduce_mean(msd, axis=0), axis=-1)
+        # msd = tf.reduce_sum(tf.reduce_mean(msd, axis=0), axis=-1)
+        msd = tf.reduce_sum(tf.reduce_sum(msd, axis=0), axis=-1)
```

Comment:
I don't know the shape here, but I assume something like `(a, b, c)` so it would be a little faster to use
`tf.reduce_sum(msd, axis=(0, -1))` or `tf.einsum("abc -> b", data)`

Reply:
I think in the calculator restructure these have all been re-written so to avoid (more than will already exist) conflicts I would leave it for the time being.